src/ui/.prettierrc

```
@@ -0,0 +1,13 @@
+{
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
```

```suggestion
  "bracketSameLine": false,
```

jsxBracketSameLine was deprecated I believe